mitodl/open-discussions
# new / upcoming courses widget on home page

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #1902 

#### What's this PR do?

This adds a new pseudo-widget to the home screen which shows the user a mix of new and upcoming courses that Open knows about.

#### How should this be manually tested?

The UI should not appear unless you have the `course_ui` feature flag on. With that on you should see on the home screen a new card on the right which shows a mix of new and upcoming courses. You should be able to click on a course to open it in a new window, and clicking the 'view more' link should take you to `/courses/`.


#### Screenshots (if appropriate)

![fnasdfnf](https://user-images.githubusercontent.com/6207644/54541722-d5953100-4970-11e9-9e32-c4d818b8aacd.png)
